### PR TITLE
Add GoFundMe presence

### LINKED ISSUE #3816: GoFundMe | gofundme.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 

**What's the name of the service?** 

https://gofundme.com | GoFundMe

**What should the Presence display?** 

Line 1: whether a user's browsing the home page, search, or a campaign page
Line 2: if in a search query, show search terms. If on a campaign page, show title of campaign page

**If possible, please provide a logo for the service (512x512 minimum)** 

https://image.flaticon.com/icons/png/512/2111/2111427.png

### PR DESCRIPTION
**Pull request by blacbrue**
Resolves #3816

<details>
<summary>Proof (expand):</summary>

<br>

Homepage
wdPmh19wWc

Search
KsM0tN3pvY

Viewing at a fundraiser
r9P1lD7oXh

</details>

### CODE REVIEW

**Comment by jaydenkoh** [comment]
#3816

File: websites/G/GoFundMe/presence.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    data.details = "Viewing homepage";
+   else if (pathname.startsWith(`/s`)) {
+      data.details = "Searching for a fundraiser:";
+      data.state = searchParams;
+  } else if (pathname.includes(`/f/${fundraiserURL}`)) {
+    const fundraiserName = document.querySelector(
+      "#root > div > main > div.p-campaign > header > h1"
+    );
+
+    data.details = "Viewing at a fundraiser:";
```
Comment (Bas950) [inline websites/G/GoFundMe/presence.ts:26]:
```suggestion
    data.details = "Viewing fundraiser:";
```

File: websites/G/GoFundMe/presence.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    data.state = fundraiserName.textContent;
+    data.buttons = [
+      {
+        label: "Visit fundraiser",
```
Comment (Bas950) [inline websites/G/GoFundMe/presence.ts:30]:
```suggestion
        label: "View fundraiser",
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — DISMISSED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Comment by jaydenkoh** [comment]
Closes #3816

**Review by Bas950 — APPROVED** [review]

**Pull request merged**